CONTRIBUTING.md

```diff
@@ -0,0 +1,87 @@
+# Contributing
+
+The following information provides a set of guidelines for contributing to the Osmo chain dev repo. Use your best judgment, and, if you see room for improvement, please propose changes to this document.
+
+The contributing guide for Osmosis explains the branching structure, how to use the SDK fork, how to make / test updates to SDK branches and how to create release notes.
+
+Contributions come in the form of writing documentation, raising issues / PRs, and any other actions that help develop the Osmo protocol documentation.
+
+## First steps
+
+The first step is to find an issue you want to fix. To identify issues we think are good for first-time contributors, we add the **good first issue** label.
+
+If you have a feature request, please use the [feature-request repo](https://github.com/osmosis-labs/feature-requests)
+
+Once you find an existing issue that you want to work on or if you have a new issue to create, continue below.
+
+## Proposing changes
+
+To contribute a change proposal, use the following workflow:
```

Comment:
Lets add making a github issue on Osmosis for anything of substance, or posting on the issue if you want to work on it